library/Zend/InputFilter/InputFilterAbstractServiceFactory.php

```diff
@@ -52,9 +49,8 @@ public function canCreateServiceWithName(ServiceLocatorInterface $inputFilters,
      * @param string                  $rName
      * @return \Zend\InputFilter\InputFilterInterface
      */
-    public function createServiceWithName(ServiceLocatorInterface $inputFilters, $cName, $rName)
```

Comment:
This is actually correct, because it is an instance of `InputFilterPluginManager` and not the `ServiceManager` itself.